tests/system-defaults-write.nix

```diff
@@ -73,24 +85,25 @@
   system.defaults.WindowManager.StandardHideWidgets = true;
   system.defaults.WindowManager.StageManagerHideWidgets = true;
   system.defaults.CustomUserPreferences = {
-      "NSGlobalDomain" = { "TISRomanSwitchState" = 1; };
-      "com.apple.Safari" = {
-        "com.apple.Safari.ContentPageGroupIdentifier.WebKit2DeveloperExtrasEnabled" =
-          true;
-      };
+    "NSGlobalDomain" = { "TISRomanSwitchState" = 1; };
+    "com.apple.Safari" = {
+      "com.apple.Safari.ContentPageGroupIdentifier.WebKit2DeveloperExtrasEnabled" =
+        true;
     };
-  test = lib.strings.concatMapStringsSep "\n" (x: ''
-    echo >&2 "checking defaults write in /${x}"
-    ${pkgs.python3}/bin/python3 <<EOL
-import sys
-from pathlib import Path
-fixture = '${./fixtures/system-defaults-write}/${x}.txt'
-out = '${config.out}/${x}'
-if Path(fixture).read_text() not in Path(out).read_text():
-  print("Did not find content from %s in %s" % (fixture, out), file=sys.stderr)
-  sys.exit(1)
-EOL
-  '') [
+  };
+  test = lib.strings.concatMapStringsSep "\n"
+    (x: ''
+      echo >&2 "checking defaults write in /${x}"
+      ${pkgs.python3}/bin/python3 <<EOL
+      import sys
```

Comment:
Was this indentation an intentional change?

Reply:
Yes, I reformatted the code with nixfmt to make it more consistent.